clang/test/CodeGen/fp-accuracy.c

```diff
@@ -30,6 +30,9 @@
 // RUN: -Wno-return-type -Wno-implicit-function-declaration -emit-llvm -o - %s \
 // RUN: | FileCheck --check-prefixes=CHECK-DEFAULT %s
 
+// Disabled due to https://github.com/intel/llvm/issues/9934
```

Comment:
Can they be disabled only for failing configuration?

Reply:
Can you suggest the configuration for linux?

Reply:
Something like `XFAIL: system-linux`.

Reply:
How we ended up committing failing test?

Reply:
@zahiraam investigating this. The failure is only reproducible with self-build, which is only in post-commit, so there were no failures in pre-commit on Linux.

Reply:
> XFAIL will fail if test passes. If the test can pass on some linux, In order to disable we probably should use REQUIRES or UNSUPPORTED instead of XFAIL.

Right, changed to `UNSUPPORTED`.

Reply:
The tests are failing on self-build on Linux and only on a release compiler. It passes with debug self-build compiler.